pillar/metalk8s/roles/master.sls

```diff
@@ -0,0 +1,38 @@
+certificates:
+  client:
+    files:
+      apiserver-etcd:
+        path: /etc/kubernetes/pki/apiserver-etcd-client.crt
+        regen_sls:
+          - metalk8s.kubernetes.apiserver
+      front-proxy:
+        path: /etc/kubernetes/pki/front-proxy-client.crt
+        regen_sls:
+          - metalk8s.kubernetes.apiserver
+      apiserver-kubelet:
+        path: /etc/kubernetes/pki/apiserver-kubelet-client.crt
+        regen_sls:
+          - metalk8s.kubernetes.apiserver
+  kubeconfig:
+    files:
+      admin:
+        path: /etc/kubernetes/admin.conf
+        cn: kubernetes-admin
+        regen_sls:
+          - metalk8s.kubernetes.apiserver.kubeconfig
```

Comment:
This is a slightly odd state name (granted, not introduced by this PR), since `admin.conf` has little to no relation to the apiserver being deployed somewhere.

Reply:
Yes, I agree, it seemed really weird to me when I saw that.
Maybe we should open an issue to tackle that once this PR is merged... or later.